cluster/src/main/java/org/apache/iotdb/cluster/utils/RaftUtils.java

```diff
@@ -302,4 +314,87 @@ public static Status createErrorStatus(String errorMsg){
     status.setCode(-1);
     return status;
   }
+
+  public static ConcurrentHashMap<String, PeerId> getGroupLeaderCache() {
+    return groupLeaderCache;
```

Comment:
please pay attention to how `groupLeaderCache` updates, you will find leader information within it sometimes is wrong

Reply:
I know, but I think here we just support local consistency, because the wrong case is rare.